bin/rails

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
+begin
+  load File.expand_path("../spring", __FILE__)
+rescue LoadError
```

Comment:
__[staging][rubocop]__
* Do not suppress exceptions. [:ref](https://github.com/bbatsov/ruby-style-guide#dont-hide-exceptions)

<!-- Comment by [SideCI](https://sideci.com) -->